ckanext/datagovtheme/templates/macros/form.html

```diff
@@ -429,37 +429,11 @@
 {% macro image_upload(data, errors, field_url='image_url', field_upload='image_upload', field_clear='clear_upload',
                       is_url=false, is_upload=false, is_upload_enabled=false, placeholder=false,
                       url_label='', upload_label='', field_name='image_url')  %}
-  {% set placeholder = placeholder if placeholder else _('http://example.com/my-image.jpg') %}
+  {% set placeholder = placeholder if placeholder else _('https://example.com/my-image.jpg') %}
   {% set url_label = url_label or _('Image URL')  %}
   {% set upload_label = upload_label or _('Image')  %}
   {% set previous_upload = data['previous_upload'] %}
 
-  {% if is_upload_enabled %}
-
-
-  <div class="image-upload"
-       data-module="image-upload"
-       data-module-is_url="{{ 'true' if is_url else 'false' }}"
-       data-module-is_upload="{{ 'true' if is_upload else 'false' }}"
-       data-module-field_url="{{ field_url }}"
-       data-module-field_upload="{{ field_upload }}"
-       data-module-field_clear="{{ field_clear }}"
-       data-module-upload_label="{{ upload_label }}"
-       data-module-field_name="{{ field_name }}"
-       data-module-previous_upload="{{ 'true' if previous_upload else 'false' }}">
-  {% endif %}
-
-
```

Comment:
Instead of customizing this, it seems like we could set `is_upload_enabled` to false somehow through ckan config?

Reply:
This is terrible configuration settings... https://github.com/ckan/ckan/blob/a7dc3e1890a331c584013a4771d52e3458d77850/ckan/lib/helpers.py#L2407-L2411

Reply:
yep, that leads to failed attempt of disabling storage-path https://github.com/GSA/catalog.data.gov/pull/775

Reply:
Who ever thought setting a path would constitute enabling a feature...?!